R/pkg/R/functions.R

```diff
@@ -2254,18 +2198,12 @@ setMethod("approxCountDistinct",
             column(jc)
           })
 
-#' Count Distinct Values
+#' @section Details:
+#' \code{countDistinct}: Returns the number of distinct items in a group.
 #'
-#' @param x Column to compute on
-#' @param ... other columns
```

Comment:
this is the example perhaps we are losing some details by grouping these functions - instead of having a line that says "other columns" now we have "other arguments" which is perhaps less clear?

thought?